### PR TITLE
Fix duplicate definition with GCC 10

### PR DESCRIPTION
**Pull request by sergeyklay**
Fixes: https://github.com/ice/framework/pull/266
Closes: https://github.com/phalcon/zephir/pull/2085

### CODE REVIEW

File: ext/test.c [hunk 1]
```diff
@@ -216,6 +216,10 @@ zend_class_entry *test_unsettest_ce;
 zend_class_entry *test_usetest_ce;
 zend_class_entry *test_vars_ce;
 
+#ifdef ZTS
+ZEND_TSRMLS_CACHE_DEFINE()
+#endif
```
Comment (sergeyklay) [inline ext/test.c:221]: @remicollet 
Shouldn't it be changed as follows?

```c
/* implement standard "stub" routine to introduce ourselves to Zend */
#ifdef COMPILE_DL_%PROJECT_UPPER%
# ifdef ZTS
ZEND_TSRMLS_CACHE_DEFINE()
# endif
ZEND_GET_MODULE(%PROJECT_LOWER_SAFE%)
#endif
``` 

and bellow

```diff
static PHP_GINIT_FUNCTION(test)
{
- #ifdef ZTS
+ #if defined(COMPILE_DL_%PROJECT_UPPER%) && defined(ZTS)
 	ZEND_TSRMLS_CACHE_UPDATE();
 #endif
```

Comment (remicollet) [reply ext/test.c:221]: indeed, even if don't see in which case `COMPILE_DL_%PROJECT_UPPER%` won't be define (excepted stupid edge case of people merge some ext in php-src)

Comment (sergeyklay) [reply ext/test.c:221]: Thank you
